test/README.md

```diff
@@ -0,0 +1,64 @@
+# Media Insights Content Localization Testing - How To
+
+### Overview
```

Comment:
Our future selves will probably thank our current selves if we instruct users how to run end-to-end tests in a Docker container. That approach will help prevent the "works on my machine" bug response scenario.

Reply:
e.g. The pupeteer tests in CAS run in Docker:
https://github.com/awslabs/aws-content-analysis/blob/development/.github/workflows/pr-workflow.yml

Reply:
I added issue #155 to track this idea.